cypress-tests-data.sh

```diff
@@ -0,0 +1,20 @@
+files="-f docker-compose.yml -f ./smoke-tests/docker-compose.cypress.yml"
+
+# author and all literature records are needed for author detail (publications & cited by), author update submission, literature search, literature detail
+# experiment (some of the literature), job, conference (some of the literature), seminar records are needed for the detail pages repectively and some for update submissions
+docker-compose $files exec web inspirehep importer records -f data/records/seminars/1799778.json -f data/records/conferences/1217045.json -f data/records/jobs/1812440.json -f data/records/authors/1274753.json -f data/records/institutions/902858.json -f data/records/experiments/1513946.json -f data/records/literature/1331798.json -f data/records/literature/1325985.json -f data/records/literature/1306493.json -f data/records/literature/1264675.json -f data/records/literature/1263659.json -f data/records/literature/1263207.json -f data/records/literature/1249881.json -f data/records/literature/1235543.json -f data/records/literature/1198168.json -f data/records/literature/1113908.json -f data/records/literature/873915.json -f data/records/literature/1688995.json -f data/records/literature/1290484.json -f data/records/literature/1264013.json -f data/records/literature/1257993.json -f data/records/literature/1310649.json -f data/records/literature/1473056.json -f data/records/literature/1358394.json -f data/records/literature/1374620.json -f data/records/literature/1452707.json -f data/records/literature/1649231.json -f data/records/literature/1297062.json -f data/records/literature/1313615.json -f data/records/literature/1597429.json -f data/records/literature/1184194.json -f data/records/literature/1322719.json -f data/records/literature/1515024.json -f data/records/literature/1510263.json -f data/records/literature/1415120.json -f data/records/literature/1400808.json -f data/records/literature/1420712.json -f data/records/literature/1492108.json -f data/records/literature/1598135.json -f data/records/literature/1306493.json -f data/records/literature/1383683.json -f data/records/literature/1238110.json
+
```

Comment:
could we delete the existing ones and just keep the new ones? Then we could run just ```inspirehep importer records -d data/records/literature``` , ```inspirehep importer records -d data/records/authors``` etc

Reply:
well I think it's better to know, which ids explicitly we need for the screenshots. That's why I didn't bother a lot.